<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qpid::InterpretationSelect</class>
 <widget class="QDialog" name="Qpid::InterpretationSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Voice Easy</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widgetRound {
	border: 1px solid #c3c3c3;
	background: #ffffff;
}

#labelSourceLanguage,#labelTargetLanguage {
	font-weight: bold;
}

#frameMain QComboBox {
	border: 1px solid #cccccc;
	padding: 0px 18px 0px 3px;
	min-height: 30px;
}

#frameMain QComboBox:on { /* shift the text when the popup opens */
	padding-top: 3px;
	padding-left: 4px;
}

#frameMain QComboBox::drop-down {
	border: 0px;
	width: 39px;
}

#frameMain QComboBox::down-arrow {
	image: url(:/images/ui/language_down_arrow.png);
}

#frameCallContainer QPushButton {
	border: 1px solid #9c9c9c;
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                 stop:0 #f5f5f5, stop:1 #dddddd);
}

#frameCallContainer QPushButton:hover {
	border: 1px solid #767676;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetRound" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="Qpid::CallInterpreterHelp" name="widgetHelp" native="true">
        <property name="minimumSize">
         <size>
          <width>434</width>
          <height>146</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="frameMain">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>210</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QLabel" name="labelSourceLanguage">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>21</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>Select source language</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxSource">
             <property name="maximumSize">
              <size>
               <width>280</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelTargetLanguage">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>21</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>Select target language</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxTarget">
             <property name="maximumSize">
              <size>
               <width>280</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frameCallContainer">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="pushButtonOK">
                <property name="minimumSize">
                 <size>
                  <width>102</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::NoFocus</enum>
                </property>
                <property name="text">
                 <string>Add</string>
                </property>
                <property name="default">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonCancel">
                <property name="minimumSize">
                 <size>
                  <width>102</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::NoFocus</enum>
                </property>
                <property name="text">
                 <string>Cancel</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>185</width>
                  <height>11</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Qpid::CallInterpreterHelp</class>
   <extends>QWidget</extends>
   <header>callinterpreterhelp.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="phone.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>Qpid::InterpretationSelect</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
